### DISCUSSION TITLE
Deploy to Deno Deploy

### DISCUSSION THREAD

**Discussion by miguelrk** [denoflow/denoflow#2] [Q&A]
I'm wondering if this can be deployed to **deno deploy** now that they do support `eval()` and `new Function()` (dynamic import statements are not yet supported though)? Or would it still fail with `Error Detail: Code generation from strings disallowed for this context` ?

**Comment by theowenyoung** [comment]
Good to know that deno deploy now support `new Function`, but dynamic import is denoflow core dependency feature,  maybe another way is to include a huge building library, then most cases don't need dynamic import modules.


I also have another project [yamlscript](https://github.com/yamlscript/yamlscript), this project allows generate dist js code from yaml file, then the generated code can be deployed to deno deploy, how do you think about it?

**Reply by miguelrk** [reply]
Thanks for the quick response!

Didn't know about `yamlscript`! Looks great! I'll take a deeper look into it...

 Looking into the repo and docs, is `yamlscript` intended as the successor of `denoflow`? Or is it simply another project with a similar scope? From what I see at first glance, the Interface is simpler?

**Comment by theowenyoung** [comment]
I see it as a successor, but it's only a proof of concept at the moment. `yamlscript` is simpler in design, but actually more extensible. The more I write, the more it looks like lisp, so I don't have the confidence to continue developing it...haha

**Reply by miguelrk** [reply]
I got that same feeling after skimming through the repo.

 I'm not very familiar with lisp, but I can indeed see many use cases for yamlscript! I will try it out

**Reply by theowenyoung** [reply]
Thanks! I remember there are still some problems to run `yamlscript` by `deno install -A https://deno.land/x/yamlscript/ys.ts`

you may need to `git clone` it, and use some commands from [`Makefile`](https://github.com/yamlscript/yamlscript/blob/main/Makefile) , like `make d`

**Reply by theowenyoung** [reply]
@miguelrk I just pushed a fix, so now you can try it by following the repo docs.

**Reply by miguelrk** [reply]
Great! I'll check it out 🚀. Thank you
